README.md

```diff
@@ -5,14 +5,22 @@
 [![Build Status][ci-img]][ci]
 [![Report Card][report-card-img]][report-card]
 
-## Status
+UberFx is a flexible, modularized framework for building robust and performant
+services. It removes boilerplate code out of your hands and lets you focus on
+your application logic.
 
-Pre-ALPHA. API Changes are **highly likely**.
+## Status
 
-## Abstract
+Alpha. Expect minor API changes and bug fixes. (Beta release coming soon)
 
-This framework is a flexible, modularized basis for building robust and
-performant services at Uber with the minimum amount of developer code.
+## What's included
+* Configuration, logging, metrics and tracing setup by default
+* Configuration provider that seamlessly merges static and dynamic config
+* Application-level as well as runtime metrics for effective monitoring
+* Context aware logging for easy debugging
+* RPC module with thrift interfaces for microservices
+* HTTP module with intelligent defaults for web applications
+* Async task module for executing async tasks durably
```

Comment:
It's in the latter half of the sentence. Would you prefer just "Durable async task module" I added the for.. part for emphasis.